Plugins/SimpleFileBrowser/Scripts/FileBrowser.cs

```diff
@@ -1771,26 +1783,35 @@ public void OnItemSelected( FileBrowserItem item, bool isDoubleClick )
 						multiSelectionPivotFileEntry = Mathf.Clamp( multiSelectionPivotFileEntry, 0, validFileEntries.Count - 1 );
 
 						selectedFileEntries.Clear();
-						selectedFileEntries.Add( item.Position );
 
-						for( int i = multiSelectionPivotFileEntry; i < item.Position; i++ )
+						for( int i = multiSelectionPivotFileEntry; i <= item.Position; i++ )
 							selectedFileEntries.Add( i );
 
-						for( int i = multiSelectionPivotFileEntry; i > item.Position; i-- )
+						for( int i = multiSelectionPivotFileEntry; i >= item.Position; i-- )
 							selectedFileEntries.Add( i );
```

Comment:
I think item.Position will be added to selectedFileEntries twice if the currently selected file is clicked with Shift key (item.Position == multiSelectionPivotFileEntry). What do you think?

Reply:
Yes, you are quite right, and I figured out that the correct solution is quite simple: just move the `Add (item.Position)` line to the last. I have amended this with the latest commit bee7e6978ff72a6d2776d25ecf870edf1110996d.